saemref/install.sls

```diff
@@ -41,3 +41,48 @@ cubicweb-create:
     - require:
         - pkg: cube-packages
         - user: {{ saemref.instance.user }}
+
+{% if saemref.instance.wsgi %}
+
+{% if not saemref.instance.cookie_secret_key %}
```

Comment:
Remove the default value for `cookie_secret_key` instead. Then this will just crash.